proximity-beacon-firmware/src/proximity_beacon_publisher.cpp

```diff
@@ -19,8 +19,9 @@ static void timer_cb(const uavcan::TimerEvent& event)
         proximity_beacon_signal_delete(pbs);
     }
 
-    /* periodic setpoint update to keep the motor running */
-    control_update_voltage_setpoint(-4.0);
+    /* Rotate at 10 Hz */
+    const auto setpoint = -10 * 2 * 3.14f;
```

Comment:
```suggestion
    const float setpoint = -10 * 2 * M_PI;
```

Reply:
M_PI is not a part of the C++ standard unfortunately...

Reply:
It works if you add

#define _USE_MATH_DEFINES
#include <math.h>

Reply:
It's sad that we have to use an internal define to use `M_PI` and there is no `math::pi<float>()` or equivalent C++-ish way of using pi. My sorrow knows no bounds 
![sad_christmas](https://user-images.githubusercontent.com/6131403/56920347-44fb5600-6ac3-11e9-8362-6a06acffddb9.gif)